system/core/base.php

```diff
@@ -482,6 +482,15 @@ function hypha_addPage($type, $language, $name, $private) {
 		return hypha_setPage($newPage, $language, $name, $private);
 	}
 
+	function hypha_deletePage($id) {
+		global $hyphaXml;
+		$hyphaXml->requireLock();
+		$targetPage = hypha_getPageById($id);
+		if ($targetPage instanceof HyphaDomElement) {
+			$hyphaXml->getElementsByTagName('pageList')->Item(0)->removeChild($targetPage);
```

Comment:
Should we *really* delete pages? Or mark them as deleted and allow an admin to view a list of deleted pages? Probably a separate discussion that should not block this merge, though.

Reply:
Good idea! This can be implemented if needed.

Reply:
I created #88 for this.